check-environment-and-install

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
add the same guard we have in our scripts

also please run this script via ``ucfa``

Reply:
fixed